cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectAssemblyOperator.java

```diff
@@ -90,6 +93,7 @@ protected void delete(Reconciliation reconciliation, Handler<AsyncResult<Void>>
         List<HasMetadata> result = new ArrayList<>();
         result.addAll(serviceOperations.list(namespace, Labels.forType(AssemblyType.CONNECT)));
         result.addAll(deploymentOperations.list(namespace, Labels.forType(AssemblyType.CONNECT)));
+        result.addAll(configMapOperations.list(namespace, Labels.forType(AssemblyType.CONNECT)));
```

Comment:
I see all those Labels.forType() and think they should be factored into a single local var (Labels is immutable, so it's OK)